### DISCUSSION TITLE
How can I chache API responses?

### DISCUSSION THREAD

**Discussion by alvarogf97** [fastapi/fastapi#7813] [Questions] [answered]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

How can I chache API responses?

There is no information about that in the docs, other frameworks such as flask cache API responses by adding decorators so, there is a similar way to do that in FastApi? If no, maybe we should work on it.

**Comment by euri10** [comment, marked as answer]
https://github.com/tiangolo/fastapi/issues/651

On Tue, Feb 25, 2020 at 9:03 AM alvaro garcia fernandez <
notifications@github.com> wrote:

> First check
>
>    - I used the GitHub search to find a similar issue and didn't find it.
>    - I searched the FastAPI documentation, with the integrated search.
>    - I already searched in Google "How to X in FastAPI" and didn't find
>    any information.
>
> Description
>
> How can I chache API responses?
>
> There is no information about that in the docs, other frameworks such as
> flask cache API responses by adding decorators so, there is a similar way
> to do that in FastApi? If no, maybe we should work on it.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tiangolo/fastapi/issues/1033?email_source=notifications&email_token=AAINSPWCCUWDAUACL2LIPY3RETGGTA5CNFSM4K3ASKG2YY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4IP7LHHA>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAINSPRIMN4MOWO3WESIWLDRETGGTANCNFSM4K3ASKGQ>
> .
>


-- 
benoit barthelet
http://pgp.mit.edu/pks/lookup?op=get&search=0xF150E01A72F6D2EE

**Comment by alvarogf97** [comment]
Thanks! Have you got an example of it?

**Comment by euri10** [comment]
yes https://github.com/argaen/aiocache/issues/463#issuecomment-534656962

**Comment by alvarogf97** [comment]
@tiangolo could you consider to add this post to the FastApi docs?

**Comment by phy25** [comment]
I mean you can definitely make a PR to the docs

**Comment by alvarogf97** [comment]
I will prepare it! thanks

**Comment by tiangolo** [comment]
Thanks everyone for the discussion here! I think you have a working solution now, right?

May we close this issue?

**Comment by alvarogf97** [comment]
I got It! Thanks everyone.
